src/test/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpointIT.java

```diff
@@ -212,32 +212,40 @@ public void shouldReturnResultsWithFilteredOutFieldsWhenRelevantCaseFieldAccessN
 
         assertEquals("Incorrect view items count", 2, searchResultViewItems.size());
 
+        assertEquals("Incorrect view items count", 2, searchResultViewItems.size());
         assertNotNull(searchResultViewItems.get(0).getCaseId());
-        assertThat(searchResultViewItems.get(0).getCaseFields().get("PersonFirstName"), is(nullValue()));
-        assertEquals("Pullen", searchResultViewItems.get(0).getCaseFields().get("PersonLastName"));
-        assertEquals("Governer House", ((Map) searchResultViewItems.get(0).getCaseFields().get("PersonAddress"))
+
+        SearchResultViewItem getFirstItem =
+            searchResultViewItems.stream().filter(e -> e.getCaseFields().get("PersonLastName").equals("Pullen")).findFirst().get();
+
```

Comment:
Can we abstract this into a method and call this findItemByLastName

getFirstItem does not really carry out the meaning

Reply:
addressed